to review

niksmo opened the pull request.

gennady-bars reviewed (commented):
Здравствуйте. (**Нужно развернуть общий комментарий ↓**)


К сожалению, работу придется отклонить без потери итерации


* В `Readme` должна быть ссылка на бриф. Эта ссылка должна быть доступной ревьюеру тоже
* При запуске проекта ничего не отображается https://disk.yandex.ru/i/eMvjNS-1JcQmOA 
* Как будто нет кода проекта в пулл-реквесте. Нужно перепроверить, что запушили изменения

niksmo commented:
> Привет! Мы всё верно запушили - страницы только-только начали собираться, до этого мы занимались компонентами.

gennady-bars commented on src/pages/layout/index.tsx:
```diff
@@ -0,0 +1,17 @@
+export function Layout() {
+  return (
+    <>
+      <header><Header/></header>
```
`      <header><Header/></header>
`


Тег `header` уже есть внутри `Header`, поэтому не нужно еще раз его указывать снаружи

gennady-bars commented on src/entities/chat/ui/chat/index.tsx:
```diff
@@ -0,0 +1,124 @@
+  const [inputValue, setInputValue] = useState<string>("");
+  const sortedMessages = sortMessages(messages);
+
+  const handleInputChange = ({ target }: ChangeEvent<HTMLInputElement>) => {
```
const handleInputChange = ({ target }: ChangeEvent<HTMLInputElement>) => {

niksmo replied:
Так не сработает 
<img width="426" alt="image" src="https://github.com/YaPomogayu/frontend/assets/96130492/f2e7b930-3e5a-415a-a603-82d5c0778a45">
<img width="559" alt="image" src="https://github.com/YaPomogayu/frontend/assets/96130492/79f9e2ba-c43f-4873-98bd-39bf5b730128">

gennady-bars replied:
> Так не сработает

Коммент неправильно скопировался у меня. Не заметил, что он такой остался


Вот это хотел

**Можно лучше**

Можно сделать универсальный **кастомный** хук для контроля любого количества инпутов в любых формах:

```js
export function useForm(inputValues={}) {
  const [values, setValues] = useState(inputValues);

  const handleChange = (event) => {
    const {value, name} = event.target;
    setValues({...values, [name]: value});
  };
  return {values, handleChange, setValues};
}

```

Этот код помещают в отдельный файл `useForm.js` в папке `hooks` и импортируют функцию туда, где нужно контролировать инпуты

И Вам не нужно будет теперь вручную создавать функции обработки инпутов и т д. Все будет в одной строчке кода:

```jsx
  const {values, handleChange, setValues} = useForm({});

```

gennady-bars reviewed (commented):
Здравствуйте. (**Нужно развернуть общий комментарий ↓**)

Посмотрите на **гитхабе** все комментарии к коду (нужно прокрутить вниз страницу там)

Работа проделана огромная:



* Отлично, что почти нет типов `any`
* Хорошо сделана часть для волонтера


но есть некоторые недочеты:



* При запуске проекта в консоли ошибка `key` https://disk.yandex.ru/i/rADNb0n6CNb0BA
* Тег `header` уже есть внутри `Header`, поэтому не нужно еще раз его указывать снаружи

gennady-bars commented on src/pages/master-admin/index.tsx:
```diff
@@ -0,0 +1,3 @@
+export function MasterAdminPage() {
+  return <div>master admin page</div>;
```
`MasterAdminPage` нужно было тоже сверстать

gennady-bars commented on src/entities/viewer/hoc/route-reducer/index.tsx:
```diff
@@ -0,0 +1,9 @@
+import { RootState } from "app/store";
+import { Navigate } from "react-router-dom";
+
+export function ProfileReducer() {
```
`ProfileReducer` нужно было назвать как-то по-другому, так как `Reducer` обычно используется для редакса.

gennady-bars commented on src/shared/ui/avatar/index.tsx:
```diff
@@ -0,0 +1,23 @@
+import classnames from "classnames";
+import styles from "./styles.module.css";
+
+interface imageProps extends ImgHTMLAttributes<HTMLImageElement> {
```
`imageProps`


Кастомные типы `typescript` принято называть в `PascalCase` (с **большой буквы**)

gennady-bars commented on src/features/create-request/ui/steps/index.tsx:
```diff
@@ -0,0 +1,57 @@
+  };
+
+  return (
+    <Portal isOpened={isPopupOpen}>
+      <OverlayingPopup isOpened={isPopupOpen}>
```
Внутри `OverlayingPopup` уже есть `Portal, поэтому вокруг не нужно еще раз создавать портал

gennady-bars commented on src/features/create-request/ui/steps/task-step/task-step.tsx:
```diff
@@ -0,0 +1,118 @@
+
+export const TaskStep = ({ tasks, isMobile }: ITaskStepProps) => {
+  const { descriptionForTask, typeOfTask } = useAppSelector(
+    (state: RootState) => state.createRequest
```
`(state: RootState)` уже не нужно будет типизировать вручную стейт, когда замените везде хук `useSelector`. Нужно будет удалить дублирование типизации

gennady-bars commented on src/features/create-request/ui/steps/index.tsx:
```diff
@@ -0,0 +1,57 @@
+  isMobile: boolean;
+}
+
+export const CreateRequest = ({
```
`CreateRequest` является компонентом реакта, а их называют в виде **существительных**, а не глаголов

gennady-bars reviewed (approved):
Поздравляю! Ваша работа принята.

Вы отлично потрудились.


* Отлично сделан `Storybook`
* Хорошая верстка страниц (хоть немного неполная)

Удачного дальнейшего обучения.


но есть некоторые недочеты: 

* Скрин https://disk.yandex.ru/i/1DXmArCZq2NFkA Выпадающее меню сливается с контентом 
* При клике на логотип нужно было сделать переход на главную, где я можно было переключиться с волонтера на админа и тд (4 типа пользователей на сайте), а то в урле приходится менять данные вручную, чтобы проверить
* Надо было реализовать фильтр https://disk.yandex.ru/i/QWXQItgIp4pVgQ (хотя бы просто он должен появляться) 
* `MasterAdminPage` нужно было тоже сверстать
* В админе нужно было моковых карточек добавить в раздел **Подтверждение/Блокировка**, чтобы показать верстку
* `ProfileReducer` нужно было назвать как-то по-другому, так как `Reducer` обычно используется для редакса. 
* Вам нужно дополнить все страницы тем, что указано в макете . Моковые данные нужно поместить в блг, контакты и тд, чтобы видеть верстку этого контента
* Кастомные типы `typescript` принято называть в `PascalCase` (с **большой буквы**)
* `(state: RootState)` уже не нужно будет типизировать вручную стейт, когда замените везде хук `useSelector`. Нужно будет удалить дублирование типизации
* `CreateRequest` является компонентом реакта, а их называют в виде **существительных**, а не глаголов

niksmo commented:
> Поздравляю! Ваша работа принята.
> 
> Вы отлично потрудились.
> 
> * Отлично сделан `Storybook`
> * Хорошая верстка страниц (хоть немного неполная)
> 
> Удачного дальнейшего обучения.
> 
> но есть некоторые недочеты:
> 
> * Скрин https://disk.yandex.ru/i/1DXmArCZq2NFkA Выпадающее меню сливается с контентом
> * При клике на логотип нужно было сделать переход на главную, где я можно было переключиться с волонтера на админа и тд (4 типа пользователей на сайте), а то в урле приходится менять данные вручную, чтобы проверить
> * Надо было реализовать фильтр https://disk.yandex.ru/i/QWXQItgIp4pVgQ (хотя бы просто он должен появляться)
> * `MasterAdminPage` нужно было тоже сверстать
> * В админе нужно было моковых карточек добавить в раздел **Подтверждение/Блокировка**, чтобы показать верстку
> * `ProfileReducer` нужно было назвать как-то по-другому, так как `Reducer` обычно используется для редакса.
> * Вам нужно дополнить все страницы тем, что указано в макете . Моковые данные нужно поместить в блг, контакты и тд, чтобы видеть верстку этого контента
> * Кастомные типы `typescript` принято называть в `PascalCase` (с **большой буквы**)
> * `(state: RootState)` уже не нужно будет типизировать вручную стейт, когда замените везде хук `useSelector`. Нужно будет удалить дублирование типизации
> * `CreateRequest` является компонентом реакта, а их называют в виде **существительных**, а не глаголов

Геннадий, спасибо за ревью! Буду вспоминать ваши советы и подсказки, вы больше других ревьюили мои работы. Спасибо и удачи!

Commit:
bug-fix: разнообразил тестовые данные для отражения во вкладках /profile/requests

Commit:
feat: роутинг на страницы /profile/requests и вложенные теперь ведет на одну страницу с табами. От пути зависит открытый таб

SergeyKukhtalevV committed:
feature: добавлено отображение кнопок в заявках в соответствии с правами администратора

Commit:
feat:
Универсализировал компонент переключателя табов для всех вариантов исполнения.
Добавил передачу счетчиков в компонент.
Дублирующие компоненты удалены

Commit:
feat:
универсализировал компонент с контентом таба.
удалил дублирующие компоненты.

Commit:
feat:
собрал получение массивов данных для табов,
сортировку этих массивов,
пересчет для счетчиков в переключателе вкладок,
фильтрация для вкладки "Не обработанные"

SergeyKukhtalevV committed:
feature: блок с подтверждение/блокировка для десктопа приведен в соответствии с макетом

SergeyKukhtalevV committed:
feature: нижняя половина side-menu для десктопа приведена в соответствии с макетом

SergeyKukhtalevV committed:
feature: позиция табов для десктопа приведена в соответствии с макетом

SergeyKukhtalevV committed:
feature: позиция формы поиска для десктопа приведена в соответствии с макетом

SergeyKukhtalevV committed:
feature: позиция информации пользователя в карточке для десктопа приведена в соответствии с макетом

Commit:
refactor: по замечания ревью

SergeyKukhtalevV committed:
feature: учел замечания

SergeyKukhtalevV committed:
feature: поправил блок кнопок действия с заявками

SergeyKukhtalevV committed:
feature: поправил типизацию прав администратора

SergeyKukhtalevV committed:
feature: убрал пропсы, убрал лишний объект из стора

SergeyKukhtalevV committed:
feature: убрал лишние стили, привел в соответствие с PP

SergeyKukhtalevV committed:
feature: заменил some на includes

Commit:
bugfix: исправлена типизация роли

Commit:
bugfix: исправлена лишняя мемоизация

Commit:
feat: номер телефона открывается штатной звонилкой

ArturKaramov committed:
fix button-with-modal

ArturKaramov committed:
add second story

AlexandrGerasimov63 committed:
Пофиксили краш по дате и привели к ISOstring

AlexandrGerasimov63 committed:
Блокировка кнопки

ulbee committed:
Пропсы TaskItem объединены в item; текст для неподтвержденного реципиента вынесен в константу; изменен сторибук

ulbee committed:
Убран isStatusActive из TaskItem; небольшие комментарии

ulbee committed:
Поправлены стили для даты

ulbee committed:
Поправлено прокидывание даты в компонент реципиента

Anton-Laid committed:
feature: add defaultavatar and renaming task-user

Anton-Laid committed:
feature: add defaultavatar and renaming task-user

Anton-Laid committed:
feature: add defaultavatar and renaming task-user

AlexandrGerasimov63 committed:
Добавил алерт при пустом адресе

AlexandrGerasimov63 committed:
Поправил текст сообщения

AlexandrGerasimov63 committed:
Поправил логику алерта в степ1

AlexandrGerasimov63 committed:
Поправил common-step

AlexandrGerasimov63 committed:
prettier

vladbarsukov committed:
change store

AlexandrGerasimov63 committed:
Подготовка к отправке объекта на сервер

ArturKaramov committed:
fix Mark props and add UserMark

Zdthns committed:
добавлена логика на кнопку редактирования

Zdthns committed:
убрала ненужный код

Zdthns committed:
добавила проверку в submit  на редактирование

Zdthns committed:
поправила title в категории

Zdthns committed:
поправила title в категории

Anton-Laid committed:
feature: у волонтера в задачах работают кнопки

Anton-Laid committed:
feature: у волонтера в задачах работают кнопки

Anton-Laid committed:
feature: у волонтера в задачах работают кнопки

Zdthns committed:
прокинула id таски  в пропсах до submit для передачи на сервер

Anton-Laid committed:
feature: работа кнопок у рецепиента

Anton-Laid committed:
feature: работа кнопок у рецепиента

Anton-Laid committed:
feature: работа кнопок у рецепиента

Anton-Laid committed:
refactor: подчистил код

Anton-Laid committed:
refactor: подчистил код

Anton-Laid committed:
refactor: подчистил код

Zdthns committed:
поправила дату

Zdthns committed:
добавила логику для отображения бессрочной таски

ArturKaramov committed:
fix buttons positions, color and refactor

ArturKaramov committed:
fix buttons positions, color and refactor

ArturKaramov committed:
fix buttons positions, color and refactor

ArturKaramov committed:
fix buttons positions, color and refactor

marussia1996 committed:
Fix/user types (#152)

* изменила типизацию таски

* написала TODO

* исправления замечаний по ревью

* add comment

* Изменения типизации пользователя

* поправила stories user-info

* id -> _id у категории

* исправила isRootSelector

* теперь главный админ опредляется по isRoot

* поправила типизацию для авторизации через вк

* исправила UserInfo чтобы отображались данные пользователя, так же вместо _id пользователя показывается vkId

* дополнила комментарий

* Изменения типизации пользователя

* поправила stories user-info

* исправила isRootSelector

* теперь главный админ опредляется по isRoot

* поправила типизацию для авторизации через вк

* исправила UserInfo чтобы отображались данные пользователя, так же вместо _id пользователя показывается vkId

* дополнила комментарий

marussia1996 committed:
Merge branch 'develop' into fix/filter-types

marussia1996 committed:
разрешение конфликтов, удаление комментариев

marussia1996 committed:
Merge pull request #150 from ya-pomogau/fix/filter-types

Fix/filter types

marussia1996 committed:
Merge branch 'fix/button-with-modal' of https://github.com/ya-pomogau/frontend into fix/button-with-modal

marussia1996 committed:
prettier

marussia1996 committed:
Merge pull request #151 from ya-pomogau/fix/button-with-modal

fix button-with-modal

marussia1996 committed:
Merge branch 'fix/map-and-mark' of https://github.com/ya-pomogau/frontend into fix/map-and-mark

marussia1996 committed:
разрешение конфликта

marussia1996 committed:
Merge pull request #156 from ya-pomogau/fix/map-and-mark

Fix/map and mark

marussia1996 committed:
проверка работы + изменения логики

marussia1996 committed:
проверка работы + изменения логики

marussia1996 committed:
проверка работы + изменения логики

marussia1996 committed:
проверка работы + изменения логики

ArturKaramov committed:
fix: conflict buttons

ArturKaramov committed:
fix: conflict buttons

ArturKaramov committed:
fix: conflict buttons

ArturKaramov committed:
fix: conflict buttons

ArturKaramov committed:
fix: button style

ArturKaramov committed:
fix: button style

ArturKaramov committed:
fix: button style

ulbee committed:
Добавлен лэйаут таски и условие показа кнопок в блоке пользователя

ulbee committed:
Добавлен лэйаут таски и условие показа кнопок в блоке пользователя

ArturKaramov committed:
fix confirm button

ArturKaramov committed:
fix confirm button

ArturKaramov committed:
fix confirm button

marussia1996 committed:
подправила отображение кнопок

marussia1996 committed:
подправила отображение кнопок

marussia1996 committed:
подправила отображение кнопок

marussia1996 committed:
подправила отображение кнопок

marussia1996 committed:
подправлено отображение бэкграунда тасок

marussia1996 committed:
подправлено отображение бэкграунда тасок

marussia1996 committed:
Merge pull request #159 from ya-pomogau/fix/task-type-props-1

Добавлен лэйаут таски и условие показа кнопок в блоке пользователя

marussia1996 committed:
Merge branch 'fix/fix-component-in-task' of https://github.com/ya-pomogau/frontend into fix/fix-component-in-task

marussia1996 committed:
изменения после решения конфликтов

marussia1996 committed:
изменения после решения конфликтов

marussia1996 committed:
Merge pull request #158 from ya-pomogau/fix/fix-component-in-task

Fix/fix component in task

marussia1996 committed:
Merge branch 'fix/task-type-props' of https://github.com/ya-pomogau/frontend into fix/task-type-props

marussia1996 committed:
правки после разрешения конфликтов

marussia1996 committed:
Merge pull request #160 from ya-pomogau/fix/task-type-props

Fix/task type props

AlexandrGerasimov63 committed:
Merge branch 'origin/fix/edit-task' of github.com:ya-pomogau/frontend into origin/fix/edit-task

AlexandrGerasimov63 committed:
Доработали редактирование

marussia1996 committed:
Merge pull request #157 from ya-pomogau/origin/fix/edit-task

Origin/fix/edit task

AlexandrGerasimov63 committed:
Запрос на создание таски

AlexandrGerasimov63 committed:
Запрос на создание таски

AlexandrGerasimov63 committed:
Запрос на создание таски

jsapro committed:
chore: временно удалил json server для работы с бэкендом из докера

jsapro committed:
chore: временно удалил json server для работы с бэкендом из докера

marussia1996 committed:
Merge branch 'feature/admins' of https://github.com/ya-pomogau/frontend into feature/admins

marussia1996 committed:
изменения типизации и тп.

SergeyKukhtalevV committed:
bugfix: вернул заявки в три столбца

jsapro committed:
Merge branch 'feature/admins' of github.com:ya-pomogau/frontend into feature/admin-actions

marussia1996 committed:
поправила типизацию таски

marussia1996 committed:
поправила отображение кнопки закрытия попапов

marussia1996 committed:
add api for get tasks | active | completed for volunteer & recipient, add slice for create task

marussia1996 committed:
add api for get tasks | active | completed for volunteer & recipient, add slice for create task

marussia1996 committed:
add api for get tasks | active | completed for volunteer & recipient, add slice for create task

marussia1996 committed:
комменты

marussia1996 committed:
комменты

marussia1996 committed:
комменты

marussia1996 committed:
изменила таску

marussia1996 committed:
решение замечаний

jsapro committed:
Revert "временно удалил скрипт запуска json-server чтобы подключить бэкенд"

This reverts commit 85b97c82747b25c14f43dc89b7803b5bfa2aae36.

jsapro committed:
Revert "временно удалил скрипт запуска json-server чтобы подключить бэкенд"

This reverts commit 85b97c82747b25c14f43dc89b7803b5bfa2aae36.

marussia1996 committed:
последняя правка db

Anton-Laid committed:
fix: решение конфликтов

Anton-Laid committed:
fix: header and db

jsapro committed:
исправлена ручка для логина администратора

jsapro committed:
исправлена ручка для логина администратора

jsapro committed:
добавлен запрос для логина администратора

jsapro committed:
добавлен запрос для логина администратора

jsapro committed:
добавлен adminLoginThunk

jsapro committed:
добавлен adminLoginThunk

jsapro committed:
добавлена логика получения токена при сабмите формы логина

jsapro committed:
добавлена логика получения токена при сабмите формы логина

marussia1996 committed:
при создании таски происходит перезапрос активных тасок

marussia1996 committed:
при создании таски происходит перезапрос активных тасок

marussia1996 committed:
при создании таски происходит перезапрос активных тасок

jsapro committed:
в слайс user добавлен adminLoginThunk

jsapro committed:
в слайс user добавлен adminLoginThunk

marussia1996 committed:
починила получение таски

marussia1996 committed:
починила получение таски

marussia1996 committed:
починила получение таски

marussia1996 committed:
---

marussia1996 committed:
---

marussia1996 committed:
Merge pull request #162 from ya-pomogau/change/task-type

Change/task type

jsapro committed:
Merge branch 'request/create-task' of github.com:ya-pomogau/frontend into feature/admin-actions

kspshnik commented on server/db.json:
```diff
@@ -277,21 +287,24 @@
-      "isPendingChanges": false
+      "volunteerReport": null,
+      "adminResolve": null,
+      "isPendingChanges": false,
```
При заполненном `recipientReport` в `isPendingChanges` должно быть `true`

kspshnik commented on server/db.json:
```diff
@@ -381,28 +408,31 @@
       "address": "Улица Парковая, 123",
       "location": [59.91776040271765, 30.323923391601433],
-      "status": "created",
+      "status": "completed",
```
Всё-таки так быть не должно :(

kspshnik reviewed (changes requested):
Проверь ещё все stories на предмет корректности указания `isPendingChanges`, и т.п.

jsapro committed:
Merge branch 'feature/admin-actions' of github.com:ya-pomogau/frontend into feature/admin-actions

Anton-Laid committed:
feature: add page conflict

marussia1996 committed:
add tags

marussia1996 committed:
add tags

marussia1996 committed:
add tags

marussia1996 committed:
Merge branch 'request/create-task' of https://github.com/ya-pomogau/frontend into request/create-task

marussia1996 committed:
добавлены запросы на подтверждения выполнения и не подтверждения у тасок и отклик на заявку

marussia1996 committed:
добавлены запросы на подтверждения выполнения и не подтверждения у тасок и отклик на заявку

marussia1996 committed:
получение с помощью токена пользователя

marussia1996 committed:
получение с помощью токена пользователя

marussia1996 committed:
добавила отображение тасок на карте неавторизованного пользовтеля

marussia1996 committed:
добавила отображение тасок на карте неавторизованного пользовтеля

marussia1996 committed:
удалили console.log

marussia1996 committed:
удалили console.log

jsapro committed:
api администратора

jsapro committed:
api администратора

marussia1996 committed:
доработала админский апи

marussia1996 committed:
доработала админский апи

marussia1996 committed:
добавила категории

marussia1996 committed:
добавила категории

marussia1996 committed:
Merge branch 'feature/admin-actions' of https://github.com/ya-pomogau/frontend into feature/admin-actions

Anton-Laid committed:
Merge branch 'feature/admin-actions' into feature/admin-chats

Anton-Laid committed:
feature: add get response

Anton-Laid committed:
Merge branch 'request/create-task' into feature/admin-chats

Anton-Laid committed:
feature: add code

Anton-Laid committed:
feature: add functional

marussia1996 committed:
правки по ревью

marussia1996 committed:
Merge pull request #166 from ya-pomogau/feature/admin-actions

Feature/admin actions

marussia1996 committed:
правки по ревью

marussia1996 committed:
правки по ревью

marussia1996 committed:
починила авторизацию через вк

marussia1996 committed:
починила авторизацию через вк

marussia1996 committed:
удалила отладочные console.log

marussia1996 committed:
удалила отладочные console.log

marussia1996 committed:
fix warning

marussia1996 committed:
fix warning

marussia1996 committed:
исправила ошибку в открытии попапа создание заявки

marussia1996 committed:
исправила ошибку в открытии попапа создание заявки

marussia1996 committed:
убрала законтированный код и console.log

marussia1996 committed:
удалила console.log

jsapro committed:
fix: добавлена дефолтная аватарка если у пользователя аватарки нет

jsapro committed:
fix: исправлено положение тултипа относительно кнопки закрытия в MainPopup

marussia1996 committed:
Merge pull request #119 from ya-pomogau/feature/admins

Feature/admins

marussia1996 committed:
Merge branch 'request/create-task' of https://github.com/ya-pomogau/frontend into request/create-task

jsapro committed:
fix: исправлен текст ошибки и стили для корректного отображения длинного текста

marussia1996 committed:
prettier

marussia1996 committed:
Merge pull request #164 from ya-pomogau/request/create-task

Request/create task

AlexandrGerasimov63 committed:
-Изменил отображение даты у бессрочной заявки
-Добавил описание для попапа неавторизованного пользователя

AlexandrGerasimov63 committed:
-Добавлено условие для подтверждения заявки
-Вернул базовые настройки центрирования карты

jsapro committed:
feature: добавлен моковый логин по строке vkId для реципиента и волонтера

jsapro committed:
fix: исправлен тип у location в типе Task

jsapro committed:
fix: исправлен location в моковых данных в stories

jsapro committed:
fix: исправлено отображение карты при редактировании таски

AlexandrGerasimov63 committed:
-Исправление ошибки 401

AlexandrGerasimov63 committed:
-Исправление ошибки 401

Anton-Laid committed:
fix: correction styles cards and scroll

jsapro committed:
fix: исправлено закрытие тултипа по любому клику внутри тултипа

jsapro committed:
fix: в ModalContent добавлены userRole и taskId для корректного формирования запроса на сервер

jsapro committed:
fix: исправлен выбор причины отмены, запрос на отмену заявки и вывод модального окна в зависимости от срока

marussia1996 committed:
Merge pull request #175 from ya-pomogau/fix/volonteer-map

FIX. Волонтёр

Anton-Laid committed:
fix: correction style and filter cards

AlexandrGerasimov63 committed:
-вынес константу токена
-добавил функции получения и добавления токена, название поменялось с token на access_token
-в userTaskApi добавил prepareHeaders, удалил лишний код
-в system-slice заменил получение токена на функцию

AlexandrGerasimov63 committed:
-вынес константу токена
-добавил функции получения и добавления токена, название поменялось с token на access_token
-в userTaskApi добавил prepareHeaders, удалил лишний код
-в system-slice заменил получение токена на функцию

AlexandrGerasimov63 committed:
Merge branch 'fix/error_401' of github.com:ya-pomogau/frontend into fix/error_401

jsapro committed:
feature: добавлен запрос на удаление таски

jsapro committed:
fix: поменял метод rejectTask на корректный метод deleteTask

jsapro committed:
Revert "revert: feature: добавлен моковый логин по строке vkId для реципиента и волонтера"

This reverts commit 0497b93b7685698ed3696a7612792141922789c8.

AlexandrGerasimov63 committed:
-Перенес константу токена
-Удалил закомментированный код

AlexandrGerasimov63 committed:
-Удалил переменную

AlexandrGerasimov63 committed:
Merge pull request #176 from ya-pomogau/fix/error_401

Исправление ошибки 401

AlexandrGerasimov63 committed:
Промежуточные изменения

AlexandrGerasimov63 committed:
Промежуточные изменения

AlexandrGerasimov63 committed:
Добавлены условия на кнопки, если у заявки нет волонтера

AlexandrGerasimov63 committed:
Добавлены условия на кнопки, если у заявки нет волонтера

AlexandrGerasimov63 committed:
Удалил консоль логи

AlexandrGerasimov63 committed:
Удалил консоль логи

AlexandrGerasimov63 committed:
-Добавил ручки для категорий
-Внес изменения в DropDown для блокировки уже выбранных категорий

AlexandrGerasimov63 committed:
Merge pull request #181 from ya-pomogau/request/categories

Request categories

AlexandrGerasimov63 committed:
Merge branch 'fix/task_btn' of github.com:ya-pomogau/frontend into fix/task_btn

INextYP committed:
Merge pull request #180 from ya-pomogau/fix/task_btn

Блокировка кнопок у реципиента, если задача бессрочна

AlexandrGerasimov63 committed:
Правки профиля

AlexandrGerasimov63 committed:
prettier

AlexandrGerasimov63 committed:
Добавил условие на кнопку

INextYP committed:
Merge remote-tracking branch 'origin/feature/admin-chats' into fix/bugs-after-testing

# Conflicts:
#	src/entities/user/model/user.ts
#	src/services/system-slice.ts

AlexandrGerasimov63 committed:
Запрос геопозиции у пользователя

AlexandrGerasimov63 committed:
Правки в условии

INextYP committed:
Merge remote-tracking branch 'origin/feature/geolocation_position' into fix/bugs-after-testing

INextYP committed:
Merge remote-tracking branch 'origin/fix/admin-pages-bags' into fix/bugs-after-testing

INextYP committed:
Merge pull request #177 from ya-pomogau/fix/admin-pages-bags

Fix/admin pages bags

INextYP committed:
Merge pull request #185 from ya-pomogau/feature/geolocation_position

Запрос геопозиции у пользователя

INextYP committed:
Merge remote-tracking branch 'origin/develop' into fix/bugs-after-testing

INextYP committed:
feat: add useAsyncAction hook, add cancel task feature; fix: bugs, cleanup code;

INextYP committed:
fix: task card view in mobile;

AlexandrGerasimov63 committed:
Перезапись токена

marussia1996 committed:
вернула функционал rejectTask

INextYP committed:
Merge remote-tracking branch 'origin/fix/recipient-tasks' into fix/bugs-after-testing

# Conflicts:
#	src/services/user-task-api.ts
#	src/widgets/task-buttons-content/index.tsx

INextYP committed:
fix: conflicts;

marussia1996 committed:
правка контактов + обновление активных тасок после запроса на отклик

marussia1996 committed:
добавлена кнопка входа через вк в мобильной версии

INextYP committed:
Merge pull request #187 from ya-pomogau/add-button-modile

Add button for modile version

INextYP committed:
Merge pull request #186 from ya-pomogau/fix/bugs-after-testing

Fix/bugs after testing

INextYP committed:
Merge pull request #184 from ya-pomogau/feature/profile_edit

Правки редактирования профиля

INextYP commented:
Пока что нет смысла в этом PR. В main будем создавать после полного функционала
